CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # CHANGELOG
 
+### [v2.4 _(Jan 11, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.5)
+
+### ✨ Highlights
+- Introducing support Installment Payments (#148)
```

Comment:
Also, usually I'll use `[#148](https://github.com/omise/omise-magento/pull/148)`

so it will be clickable at https://github.com/omise/omise-magento/blob/17bc0a02a9eb7a98a6a5a14ddd4a54c8a84e96c7/CHANGELOG.md

Reply:
it is still clickable.. github does it aoutamatically @guzzilar

Reply:
No? https://github.com/omise/omise-magento/blob/changed-user-agent-suffix-for-upcoming-release/CHANGELOG.md

Reply:
@jacstn From the link https://github.com/omise/omise-magento/blob/changed-user-agent-suffix-for-upcoming-release/CHANGELOG.md, it's not clickable by itself? (or which place you are referring with?)

Reply:
yes.. my mistake.. I was looking at my `Release` notes.. instead of changelog